cloudify_agent/shell/commands/configure.py

```diff
@@ -42,24 +42,27 @@
               help='Indication whether sudo should be used when applying '
                    ' disable-requiretty part',
               is_flag=True)
+@click.option('--user',
+              help='(For disable-requiretty) The user for which to disable '
+                   'requiretty')
 @handle_failures
-def configure(disable_requiretty, relocated_env, no_sudo):
+def configure(disable_requiretty, relocated_env, no_sudo, user):
```

Comment:
If I remember correctly, this function is used both to disable TTY and to fix the venv, but you've changed only one invocation of it. Be sure to fix the other one as well.

Reply:
`--user` is optional (see `required=False`).